low-level/framework/rabbitmq/rabbitmq_ingress_processor.py

```diff
@@ -124,35 +106,45 @@ def initialize(self, conf_reader, msgQlist, product):
         # Initialize internal message queues for this module
         super(RabbitMQingressProcessor, self).initialize_msgQ(msgQlist)
 
-        # Configure RabbitMQ Exchange to receive messages
-        self._configure_exchange(retry=False)
-
-        # Display values used to configure pika from the config file
-        self._log_debug("RabbitMQ user: %s" % self._username)
-        self._log_debug("RabbitMQ exchange: %s, routing_key: %s, vhost: %s" %
-                        (self._exchange_name, self._routing_key, self._virtual_host))
+        self._read_config()
+        producer_initialized.wait()
```

Comment:
this is the workaround i believe to avoid getting trapped due to message bus bug

Reply:
Yes